pom.xml

```diff
@@ -1677,6 +1677,11 @@
             <artifactId>fmt-maven-plugin</artifactId>
             <version>2.4.0</version>
         </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-checkstyle-plugin</artifactId>
+          <version>3.1.1</version>
```

Comment:
Maybe its worth to use a property for version of plugin maven-checkstyle-plugin as well?

Reply:
we might still have to do that as some versions of Maven don't carry over plugin versions from the default `pluginManagement`